config.yaml.example

```diff
@@ -5,12 +5,14 @@ inverters:
   serial: 5555555555
   datalog: 2222222222
   heartbeats: false
+  broadcast_holding_registers: false
```

Comment:
Per-inverter config makes natural sense when we're conditioning this on inverter connect.

I called it `broadcast` rather than `publish` to avoid mqtt terminology here, but it might not actually matter. Pretty agnostic about the naming.

Reply:
Hm, no mention of it being on startup only here though?

Reply:
Well, it's no longer on startup, it's on inverter (re)connect.

`broadcast_holding_registers_on_connect` is fine too, it's just... long.

Reply:
Well, `publish` was used in `publish_individual_input` and this is really doing pretty similar so I'm not too worried about using publish here. Maybe `publish_holdings_on_connect` ? I think having `registers` in the name is a bit redundant so can be dropped.

Reply:
OK, renamed :thumbsup:.